redis/paymentCache.ts

```diff
@@ -32,6 +32,65 @@ export const getPaymentList = async (userId: string): Promise<Payment[]> => {
   return await getCachedPaymentsForUser(userId)
 }
 
+export const getCachedPaymentsForUserWithPagination = async (
```

Comment:
This is also reminiscent of the earlier stages of this PR; being created not to be used anywhere.